samples/internat/Info.plist.in

```diff
@@ -9,11 +9,26 @@
 	<key>CFBundleGetInfoString</key>
 	<string>${MACOSX_BUNDLE_INFO_STRING}</string>
 	<key>CFBundleIconFile</key>
-	<string>${MACOSX_BUNDLE_ICON_FILE}</string>
+	<string>wxmac.icns</string>
```

Comment:
actually using the CMake var and replacing it perhaps is better than hardcoding ?

Reply:
Same as above, this has to be hard-coded for now.